engine/js/ng/shared/action-types.js

```diff
@@ -0,0 +1,58 @@
+(function(window, document) {
+  'use strict';
+
+  window.ActionTypes = {
+    GET_STATE: 'GET_STATE',
+    SET_STATE: 'SET_STATE',
+
+    CREATE_SITE: 'CREATE_SITE',
+    SITE_CREATED: 'SITE_CREATED',
```

Comment:
As discussed, let's update the redux action naming as follows:
- use `INIT` prefix for actions before sync (for loading state) - `INIT_CREATE_SITE`
- use normal action name for modifications in reducer (after sync) - `CREATE_SITE`